source/uwp/SharedRenderer/lib/Util.cpp

```diff
@@ -481,25 +481,27 @@ bool MeetsRequirements(winrt::IAdaptiveCardElement const& cardElement, winrt::Ad
     winrt::IVector<winrt::AdaptiveRequirement> requirements = cardElement.Requirements();
     bool meetsRequirementsLocal = true;
 
-    for (auto req : requirements)
+    if (requirements)
```

Comment:
not blocking, but can you see when this variable is null? I don't think it should ever be null... even if it's called on custom input it should just be of size 0. If that's an option, can we simply initialize it to empty vector where it supposed to be initialized?:)
(feel free to skip)

Reply:
I'm not sure why this variable would be null. The for-each loop would have worked fine if the size was 0. Maybe it has something to do with the host config?

We could update the object model to return an empty list. I'll merge this PR for now to catch this error.